css/selectors/descendant.json

```diff
@@ -87,7 +87,8 @@
                 "version_added": false
               },
               "safari": {
-                "version_added": "10"
+                "version_added": "10",
+                "version_removed": "11.1"
               },
               "safari_ios": {
                 "version_added": "10"
```

Comment:
Safari for iOS has also dropped support for this.

Reply:
Good catch @ExE-Boss !